pwa/components/common/Layout.tsx

```diff
@@ -41,7 +42,9 @@ const Layout = ({
             <A2HSIOS />
             <Notifications />
             <SearchRepairerProvider>
-              <UserFormProvider>{children}</UserFormProvider>
+              <DashboardRepairerProvider>
```

Comment:
I prefer that you use this context only on the pages where you need it, else it will be available and run in all the application, website, admin, sradmin.
Waiting for work with layouts from nextjs14 and app router, and to avoid to think about using a `getLayout` in a lot of pages, you can remove its invocation here then wrap only the components pages where you need to use it.
Example in all the pages that use the `DashboardLayout`: 
in pwa/pages/sradmin/boutiques/[repairer_id]/agenda/index.tsx
the page component use the `DashboardLayout` that use the context and the `AgendaCalendar` use `ModalShowApointment` that use the context too so wrap the content page in your context: 
```
  return (
    <>
      <Head>
        <title>Agenda | Rustine Libre</title>
      </Head>
      <DashboardRepairerProvider>
           <DashboardLayout>
                <Box component="main" maxWidth="1200">
                      {repairer && <AgendaCalendar repairer={repairer} />}
                </Box>
            </DashboardLayout>
      <DashboardRepairerProvider>
      ...
  ```
  
  make the same for other pages that uses it.

Reply:
Because I understand that `DashboardLayout` needs to use your context

Reply:
As we said, let's leave the context in the general layout for now